packages/rum/src/domain/segmentCollection/segment.spec.ts

```diff
@@ -14,7 +14,7 @@ const ENCODED_SEGMENT_HEADER_BYTES_COUNT = 12 // {"records":[
 const ENCODED_RECORD_BYTES_COUNT = 25
 const ENCODED_FULL_SNAPSHOT_RECORD_BYTES_COUNT = 35
 const ENCODED_SEPARATOR_BYTES_COUNT = 1 // ,
-const ENCODED_META_BYTES_COUNT = 173 // this should stay accurate as long as less than 10 records are added
+const ENCODED_META_BYTES_COUNT = 192 // this should stay accurate as long as less than 10 records are added
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** I am curious to know why this changed?

Reply:
Because we've added the `source: 'browser'` property to the segment metadata